src/redist/targets/Crossgen.targets

```diff
@@ -146,6 +146,5 @@
 
     <Exec Command="find $(SdkOutputDirectory) -type d -exec chmod 755 {} \;" />
     <Exec Command="find $(SdkOutputDirectory) -type f -exec chmod 644 {} \;" />
-    <Chmod Mode="755" Glob="$(SdkOutputDirectory)/FSharp/RunFsc.sh" />
```

Comment:
@KevinRansom double check with you, we could remove this chmod right? per Microsoft/visualfsharp#5882

Reply:
@wli3 yes. I confirm removing this is fine and necessary.